tests/lib/eslint/flat-eslint.js

```diff
@@ -3707,6 +3725,46 @@ describe("FlatESLint", () => {
                 nodePlugin.rules["no-new-require"].meta
             );
         });
+
+        it("should ignore messages not related to a rule", async () => {
+            const engine = new FlatESLint({
+                overrideConfigFile: true,
+                overrideConfig: { rules: { "no-var": "warn" } },
+                reportUnusedDisableDirectives: "warn"
+            });
+
+            {
+                const results = await engine.lintText("syntax error");
+                const rulesMeta = engine.getRulesMetaForResults(results);
+
+                assert.deepStrictEqual(rulesMeta, {});
+            }
+            {
+                const results = await engine.lintText("// eslint-disable-line no-var");
+                const rulesMeta = engine.getRulesMetaForResults(results);
+
+                assert.deepStrictEqual(rulesMeta, {});
+            }
+            {
+                const results = await engine.lintText("", { filePath: "/.ignored.js", warnIgnored: true });
```

Comment:
Can we use some explicitly ignored file path instead of `/.ignored.js`? `/.ignored.js` will indeed most likely be treated as an ignored file due to being outside of `cwd`, but that still depends on how the environment where the tests are run is set up. It may also be ignored because it's a dotfile, but it is not yet certain how dotfiles will be treated. For the purposes of this test, I think it would be better to avoid these assumptions. We can, for example, add `ignorePatterns: ["ignored.js"]` to `new FlatESLint({ ... })` and use `filePath: "ignored.js"`.

(the same for the `eslint.js` test)

Reply:
Updated as suggested, thanks!